src/99-appendices/10-file-collections.md

```diff
@@ -37,11 +43,18 @@ included in this appendix:
+| TB1AFI  | Please see the [qMRI appendix](../99-appendices/11-qmri.md). | Actual Flip Angle Imaging (AFI)    | This method ([Yarnykh 2007](https://dx.doi.org/10.1002/mrm.21120)) calculates a B1<sup>+</sup> map from two images acquired at interleaved (two) TRs with identical RF pulses using a steady-state sequence                                                                                                                                                                                                                                                                      |
+| TB1TFL  | Please see the [qMRI appendix](../99-appendices/11-qmri.md). | Siemens `tfl_b1_map`               | B1<sup>+</sup> data acquired using `tfl_b1_map` product sequence by Siemens based on the method by [Chung et al. (2010)](https://onlinelibrary.wiley.com/doi/full/10.1002/mrm.22423). The sequence generates one anatomical image and one scaled flip angle map.                                                                                                                                                                                                                 |
+| TB1RMF  | Please see the [qMRI appendix](../99-appendices/11-qmri.md). | Siemens `rf_map`                   | B1<sup>+</sup> data acquired using `rf_map` product sequence by Siemens                                                                                                                                                                                                                                                                                                                                                                                                          |
+| TB1SRGE | `flip`, `inv`                                                | SA2RAGE                            | Saturation-prepared with 2 rapid gradient echoes (SA2RAGE) uses a ratio of two saturation recovery images with different time delays, and a simulated look-up table to estimate B1+ ([Eggenschwiler et al. 2011](https://onlinelibrary.wiley.com/doi/full/10.1002/mrm.23145)). This sequence can also be used in conjunction with MP2RAGE T1 mapping to iteratively improve B1+ and T1 map estimation ([Marques & Gruetter 2013](https://doi.org/10.1371/journal.pone.0069294)). |
+| RB1COR  | Please see the [qMRI appendix](../99-appendices/11-qmri.md)  | B1<sup>-</sup> field correction    | Low resolution images acquired by the body coil (in the gantry of the scanner) and the head coil using identical acquisition parameters to generate a combined sensitivity map as described in [Papp et al. (2016)](https://onlinelibrary.wiley.com/doi/full/10.1002/mrm.26058)                                                                                                                                                                                                  |
```

Comment:
This time there is no punctuation mark after `Please see the [qMRI appendix](../99-appendices/11-qmri.md) `, but in previous table rows, there is --> please adjust for consistency :-)

Reply:
8e22a7f